### ISSUE TITLE
Close section for better admin overview

### ISSUE THREAD

**Issue by rogoit** [sulu/sulu#5094]
# Problem description

When you have long sections, like a time plan of a conference, in a admin page view it would be good to hide the content by click.

# Proposed solution

A accordion effect on a little plus minus item.

![Bildschirmfoto von 2020-03-07 01-01-56](https://user-images.githubusercontent.com/3101439/76131584-4e245000-600f-11ea-9f46-638a2cecd015.png)

**Comment by danrot** [comment]
@sulu/core-team WDYT? I think I would be OK with this, but by default the sections should be opened. However, not sure how helpful this feature is when all blocks are expanded by default...

**Comment by rogoit** [comment]
Hi Daniel,

thx for you answer. I want to close a section to have a better overview. Its just about a new button. As you see in example very long sections will make a lot of scrll word.

See you
Roland

**Comment by niklasnatter** [comment]
If there is an appealing design, this feature should not hurt anybody and might be useful for big forms. So i am in 🙂

**Comment by luca-rath** [comment]
What about introducing a param on the section which defines the default collapsed state for that section? Because probably the developer already knows which sections should be collapsed by default and which should not, right?

**Comment by rogoit** [comment]
Hi, I think a "collapse all" by default would be good and than that you can "open all" and this on top of the section

**Comment by alexander-schranz** [comment]
I know this feature from our old CMS and I must say I was never a big fan of it from content management perspective. I always like it more to have them open and scroll to them as have todo additional clicks to have things open. From content management view I don't see much usage for it and would not recommend to use it there as the content manager should need to have as less clicks as possible to edit things. But I see that there could be usage in the data management part when you have complex forms and complex domain models. I think here the decision should be on @chirimoya and we would need a design for it from our UX Team.

**Comment by luca-rath** [comment]
I would definitely not collapse all sections by default as @rogoit suggested, because in most cases you want the sections to be always opened

**Comment by CapsE** [comment]
I agree with @luca-rath the current behaivour should stay the same. Still it would be nice to be able to do something like

`<section name="Links" foldable="true" default-state="closed">`

In my case social-media links are at the top of the page so this is where I'd like to have the section to manage them. But that section also takes at least one screen height of space so the editor has to scroll quite a lot to get the section where they can actually edit the page.

**Comment by FabianKostenzer** [comment]
@alexander-schranz

Is there any update on this topic?
I think being able to "hide" groups of properties via the backend ui is a necessary feature for working on large and more complex entities/pages/articles etc.

An alternative solution to collapsible sections would be some kind of tab-navigation, which allows the editor to only see the necessary fields, he is working on, and switch to other fields in another tab if required. (Maybe something like this already exsists in Sulu. If so i was not able to find it documented anywhere).

I think, this would improve the editing experience greatly.

**Comment by alexander-schranz** [comment]
@FabianKostenzer if you work on custom entities. I would more suggest you to use multiple tabs like the page is splitted into content, seo, excerpt, settings, ... .

For pages, articles we have not yet a solution, its also hard to decide as what is not important for content manager A is maybe very important for content manager B. So personally not a fan of hide by default something, which make also searching for specific content harder (I remember that from the previous CMS). But it sure not my decision, the UX Designers was introduced to the request and I will update as soon as it gives some news. Currently we have on dev side whole focus on the new 3.0 version.
